api/cloudcontroller/ccv3/service_instance.go

```diff
@@ -0,0 +1,39 @@
+package ccv3
+
+import (
+	"net/url"
+
+	"code.cloudfoundry.org/cli/api/cloudcontroller/ccerror"
+	"code.cloudfoundry.org/cli/api/cloudcontroller/ccv3/internal"
+)
+
+type ServiceInstance struct {
+	GUID string `json:"guid"`
+	Name string `json:"name"`
+}
+
+// GetServiceInstances lists ServiceInstances with optional filters.
+func (client *Client) GetServiceInstances(query url.Values) ([]ServiceInstance, Warnings, error) {
```

Comment:
please add unit tests for this method

Reply:
😱 Not sure how this happened. Added now.